linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -42,6 +42,11 @@
     - include_tasks: ubuntu/reconfigure_ubuntu_iso_vm.yml
       when: guest_id == 'ubuntu64Guest'
 
+    # Increase VM's video card memory to 16 MB in case desktop can't be loaded
```

Comment:
Is this a workaround for an existing bug?

Reply:
RHEL with GUI will show black screen if the video card memory is default 4 MB. Yan suggested raising it to 16MB.

Reply:
RHEL 8.x and 9.x GUI works with 8MB video memory, so change it to 8MB